page-futuremakers.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+$context = Timber::get_context();
+$post = new TimberPost();
+$context['post'] = $post;
+$context['broken_content'] = explode("<!--more-->", $post->content);
+
+$context['futuremakersImages'] = [ ['src'=>'../wp-content/themes/cd-theme/static/img/languages/Flag_of_Czech_Republic.png', 'alt'=>'Alt desc 2',
```

Comment:
So, I'm all for deleting content from our theme, buuuuuut these are still referencing images within the theme. The images you just deleted. 😄